src/Model/Command/UserMerge.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+/**
+ * Merge two users. DELETES source user!
+ */
+class UserMerge extends AbstractCommand
+{
+    /** @var string */
+    private $sourceUserId;
+    /** @var string */
+    private $targetUserId;
+
+    /**
+     * Merge source user to target user AND DELETE SOURCE USER.
+     */
+    public function __construct(string $sourceUserId, string $targetUserId)
```

Comment:
All other Commands (ok, both of them :-D) could be instantiated only through the static constructor. My idea was to not mix two approaches (new vs static constructor) and even though some commands do need just one way of instantiation (thus the `::create()` method), I would prefer to make the static constructors the only way of Command instantiation (and make the `__construct()` private), so there is less cognitive load for the developer and also the usage is unified.